bot/code_review_bot/report/base.py

```diff
@@ -97,27 +80,38 @@ def requires(self, configuration, *keys):
     def calc_stats(self, issues):
         """
         Calc stats about issues:
-        * group issues by class name
+        * group issues by analyzer
         * count their total number
         * count their publishable number
         """
-        groups = itertools.groupby(
-            sorted(issues, key=lambda x: str(x.__class__)), lambda x: x.__class__
-        )
 
-        def stats(items):
+        def by_analyzer(i):
+            return i.analyzer
+
+        groups = itertools.groupby(sorted(issues, key=by_analyzer), by_analyzer)
```

Comment:
Nit: use a lambda

Reply:
At first i made a shared lambda, but :black_large_square: does not allow that.